### PR TITLE
[Lit-Mobx-Vaadin] Add README and Website Listing

### LINKED ISSUE #1224: [Lit-Mobx-Vaadin] Write the README

### PR DESCRIPTION
**Pull request by Amdrel**
## Type of change

- [x] Documentation change
- [ ] Bug fix

## Summary of change

Adds the README for the Lit starter and adds the listing to the starter.dev website.

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #1224 and #1259
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: packages/website/src/config.tsx [hunk 1]
```diff
@@ -385,8 +388,26 @@ export const TECHNOLOGIES = [
     key: 'prisma',
     name: 'Prisma',
     tags: ['Data Management'],
-    Icon: (props) => <PrismaIcon {...props} />
-  }
+    Icon: (props) => <PrismaIcon {...props} />,
+  },
+  {
+    key: 'litelement',
+    name: 'LitElement',
```
Comment (flakolefluk) [inline packages/website/src/config.tsx:395]: shouldn't it be Lit?

File: starter-kits.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "expo-zustand-styled-component": "Expo, Zustand, and Styled Components",
   "express-apollo-prisma": "Express.js, Apollo Server, and Prisma",
   "express-typeorm-postgres": "Express.js, TypeOrm, and PostgreSQL",
+  "lit-mobx-vaadin": "LitElement, Mobx, and Vaadin",
```
Comment (flakolefluk) [inline starter-kits.json:9]: Lit or LitElement?

File: starters/lit-mobx-vaadin/README.md [hunk 3]
```diff
@@ -1,22 +1,79 @@
...
+- Copy and rename the `starters/lit-mobx-vaadin` directory to the name of your new project.
+- `cd` into your project directory and run `yarn`.
+- Run `yarn start` to start the development server.
+- Open your browser to `http://localhost:8000` to see the included example code running.
 
-## Tooling configs
+## Commands
 
-For most of the tools, the configuration is in the `package.json` to reduce the amount of files in your project.
+- `yarn start` or `yarn dev` runs your app for development, reloading on file changes
```
Comment (flakolefluk) [inline starters/lit-mobx-vaadin/README.md:73]: is yarn required?

**Review by flakolefluk — CHANGES_REQUESTED** [review]
i believe latest docs use Lit for the library vs LitElement for the base class.

File: starters/lit-mobx-vaadin/README.md (hunk 3 above)
Comment (Amdrel) [reply starters/lit-mobx-vaadin/README.md:73]: It's not required. It seems like the starters inconsistently suggest using either yarn or npm.

File: starter-kits.json (hunk 2 above)
Comment (Amdrel) [reply starter-kits.json]: I changed all occurrences of LitElement to Lit 😄

**Review by flakolefluk — APPROVED** [review]

**Pull request merged**